__tests__/app.test.js

```diff
@@ -131,6 +131,58 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with array of comments for the article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+        expect(body.comments.length).toBeGreaterThan(0);
```

Comment:
As the data is reseeded beforeEach test you should be specific about the number of comments for the article